src/language/__tests__/lexer-test.js

```diff
@@ -359,6 +365,24 @@ describe('Lexer', () => {
     });
   });
 
+  it('advance line after lexing block string', () => {
+    expect(
+      lexSecond(`"""
+
+        spans
+          multiple
+            lines
+
+        """ second_token`),
```

Comment:
To cover all cases in one test you can do something like this:
```js
const str = [
  '""" \n',
  'spans \r\n',
  'multiple \n\r',
  'lines \n\n',
  '""" second_token'
].join('')
```

Reply:
This case will not cover edge cases because various combinations can annihilate

Reply:
Added, exactly like you suggested